add asdf as an installation method (#282)
grain-lang/grain-lang.org: src/getting_grain.md

You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.

> ## Alternatives

Reviewer: I'm wondering if we should call this 
```suggestion
## Community
```

And then add a description about community run installation mechanisms and that the Grain team doesn't make sure they are up-to-date.

Reviewer 2: How about this as a description?

> These installation methods are maintained by the community—they're not endorsed or supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

Reviewer: I think, just by being on the website, they are "endorsed" in a way. Let's just say "they're not supplied by the Grain team, but"

> ### Using ASDF

Author: ```suggestion
### Using asdf
```

> ## Community
> 
> ### Using asdf

Reviewer: ```suggestion
## Community

These installation methods are maintained by the community—they're not supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

### Using asdf
```

Author: looks great

You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.

## Community

These installation methods are maintained by the community—they're not supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

### Using asdf

You can install specific version of Grain toolchain using [asdf](https://asdf-vm.com) runtime version manager.

```bash
asdf plugin add grain

# List all available version
asdf list all grain

asdf install grain 0.4.6
asdf global grain 0.4.6
```

> See the [plugin](https://github.com/cometkim/asdf-grain) for detail.

Reviewer: ```suggestion
See the [plugin](https://github.com/cometkim/asdf-grain) for more details.
```

Reviewer: Just going to commit this and approve
